bin/tfr-read

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+#  Copyright 2018 Spotify AB.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+
+import argparse
+import os
+
+import tensorflow as tf
+from tensorflow.python.lib.io import file_io
+
+from spotify_tensorflow.example_decoders import ExampleWithFeatureSpecDecoder, ExampleDecoder
+from spotify_tensorflow.tf_schema_utils import SchemaToFeatureSpec
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description="Output TFRecords as JSON")
+    parser.add_argument("-s", "--schema", help="Path to Schema protobuf file. Uses Example if not "
+                                               "supplied.")
+    parser.add_argument('tf_records_paths',
+                        metavar='TF_RECORDS_PATH',
+                        nargs='+',
+                        help="TFRecords file (or directory containing .tfrecords files)")
+    return parser.parse_args()
+
+
+def list_tf_records(paths, schema):
+    for p in paths:
+        if file_io.is_directory(p):
+            local_schema_path = os.path.join(p, "_schema.pb")
```

Comment:
Should we have log a message here that say something about schema being overridden?

Reply:
seems like some fancy feature